releasenotes/README.md

```diff
@@ -36,11 +36,44 @@ ANNOUNCEMENTS:
+* NEW: Execute DevOps Center actions at the command line with these new beta CLI commands:
 
-* FIX: Salesforce DX projects now support the FundraisingConfig metadata type. 
+    * `project deploy pipeline start`: Deploy changes from a branch to the pipeline stage’s org.
+    * `project deploy pipeline report`: Check the status of a pipeline deploy operation.
+    * `project deploy pipeline validate`: Perform a validate-only deployment from a branch to the pipeline stage’s org. 
+    * `project deploy pipeline quick` : Quickly deploy a validated deployment to an org.
+    * `project deploy pipeline resume`: Resume watching a pipeline deploy operation.
+
+    We created these commands for developers who want to work outside of DevOps Center or want to automate tasks. The commands are in [plugin-devops-center](https://github.com/salesforcecli/plugin-devops-center), which is a just-in-time (JIT) plugin. After you update to this Salesforce CLI release, run a DevOps Center command in a terminal or command window and the CLI automatically installs the plugin and then runs the command. Nifty, huh. Let's look at a few examples to see how the new beta commands work.
+  
+    In this example, several work item feature branches were previously merged to the `recruit-integration` branch. To deploy the changes from the `recruit-integration` branch to its associated Integration environment:
+
+    ```bash
+    sf project deploy pipeline start --branch-name recruit-integration --devops-center-username MyDevOpsCenterOrg --devops-center-project-name "Recruiting App"
+    ```
+
+    In this example, UAT is the bundling stage, which means that you must indicate a version identifier with the `--bundle-version-name` flag. To deploy all merged changes in a version 1.0 bundle to the UAT environment: 
+
+    ```bash
+    sf project deploy pipeline start --branch-name recruit-uat --devops-center-username MyDevOpsCenterOrg --devops-center-project-name "Recruiting App" --bundle-version-name 1.0
+    ```
+
+    In this example, you first validate the changes in the `recruit-staging` branch so you can later perform a quick deployment to production. The validate command returns a job ID that you later pass to the quick deploy command.
+
+    ```bash
+    sf project deploy pipeline validate --branch-name recruit-staging --devops-center-username MyDevOpsCenterOrg --test-level RunLocalTests --devops-center-project-name "Recruiting App"
+    ```
+
+    When ready, execute the quick deployment to production by specifying the job ID returned by the validate command:
+
+    ```bash
+    sf project deploy pipeline quick --job-id 0Af0x000017yLUFCA2
+    ```
+
+    As always, run the commands with the `--help` flag to see more examples and information. Enjoy!
```

Comment:
@jshackell-sfdc This looks great.
Could we also mention that we are introducing the new configuration variable `target-devops-center`?

Reply:
Yep, I'll add it now. Thx!